crates/ironrdp-cliprdr/src/backend/windows.rs

```diff
@@ -0,0 +1,165 @@
+mod clipboard_data;
+mod clipboard_data_ref;
+mod clipboard_impl;
+mod cliprdr_backend;
+mod os_clipboard;
+mod remote_format_registry;
+
+use self::clipboard_impl::{clipboard_subproc, WinClipboardImpl};
+use self::cliprdr_backend::WinCliprdrBackend;
+
+use crate::backend::{ClipboardMessageProxy, CliprdrBackendFactory};
+use crate::pdu::{
+    ClipboardFormat, ClipboardFormatId, ClipboardGeneralCapabilityFlags, FormatDataRequest, FormatDataResponse,
+};
+use crate::CliprdrBackend;
+
+use thiserror::Error;
+use tracing::error;
+use winapi::shared::basetsd::DWORD_PTR;
+use winapi::shared::windef::HWND;
+use winapi::um::commctrl::{RemoveWindowSubclass, SetWindowSubclass};
+use winapi::um::winuser::{AddClipboardFormatListener, RemoveClipboardFormatListener, WM_USER};
+
+use std::sync::{mpsc as mpsc_sync, Arc};
+
+const BACKEND_CHANNEL_SIZE: usize = 8;
+const WM_USER_CLIPBOARD: u32 = WM_USER;
```

Comment:
I think this needs to be either more explicit, or to not exist at all (WM_USER is not much different and is the "official" name)
```suggestion
const WM_CLIPRDR_BACKEND_EVENT: u32 = WM_USER;
```